### PR TITLE
Adding new lumi and bkg norm values

### PR DESCRIPTION
**Pull request by vukasinmilosevic**

### CODE REVIEW

File: Inputs/Input_Info.py [hunk 1]
```diff
@@ -11,9 +11,9 @@
 INPUT_m4l_high = 160.0
 
 # Lumi for each year
-Lumi_2016 = 36.6
-Lumi_2017 = 41.5
-Lumi_2018 = 59.7
+Lumi_2016 = 36.31
```
Comment (ram1123) [inline Inputs/Input_Info.py]: 2016 the lumi should be 36.33.

19.52 + 16.81 = 36.33

https://docs.google.com/presentation/d/1gEF2jExqS1hWs-dlYgxrTT2xQztYbBZcrLdw45Aiy7g/edit#slide=id.ge8cc301637_0_55

**Comment by ram1123** [comment]
More recent yaml file is grabbed from Tahir PR:

https://github.com/ram1123/Fiducial_XS/pull/6

Updated lumi value by my hand in `Input_Info.py`

**Comment by ram1123** [comment]
By mistake, it was closed.

As this has the updated Zjets numbers from LLR. So, resolved the conflict and updated the 2016 lumi number as per the latest recommendataion.

Now, I will merge this.

File: Inputs/Input_Info.py [hunk 2]
```diff
@@ -11,9 +11,9 @@
 INPUT_m4l_high = 160.0
 
 # Lumi for each year
-Lumi_2016 = 36.6
-Lumi_2017 = 41.5
-Lumi_2018 = 59.7
+Lumi_2016 = 36.33
```
Comment (vukasinmilosevic) [inline Inputs/Input_Info.py:14]: I still see .33 instead of .31 no? @ram1123

Comment (ram1123) [reply Inputs/Input_Info.py:14]: 0.33 is the latest number. Earlier it was 36.31 I updated this to 36.33

Below attaching the screenshot :

Screenshot 2022-08-25 at 18 20 23

**Pull request merged**